addons/mail/static/src/core/web/store_service_patch.js

```diff
@@ -47,15 +46,29 @@ const StorePatch = {
             model: "mail.box",
             name: _t("History"),
         };
-        try {
-            // useful for synchronizing activity data between multiple tabs
-            this.activityBroadcastChannel = new browser.BroadcastChannel("mail.activity.channel");
-            this.activityBroadcastChannel.onmessage =
-                this._onActivityBroadcastChannelMessage.bind(this);
-        } catch {
-            // BroadcastChannel API is not supported (e.g. Safari < 15.4), so disabling it.
-            this.activityBroadcastChannel = null;
-        }
+        this.env.services["multi_tab"].bus.addEventListener(
+            "mail.activity/insert",
+            ({ detail }) => {
+                this.store.Activity.insert(detail, { broadcast: false, html: true });
+            }
+        );
+        this.env.services["multi_tab"].bus.addEventListener(
+            "mail.activity/delete",
+            ({ detail }) => {
+                const activity = this.store.Activity.insert(detail, { broadcast: false });
+                activity.delete({ broadcast: false });
+            }
+        );
+        this.env.services["multi_tab"].bus.addEventListener(
+            "mail.activity/reload_chatter",
+            ({ detail }) => {
+                const thread = this.store.Thread.insert({
+                    model: detail.model,
+                    id: detail.id,
+                });
+                thread.fetchNewMessages();
+            }
+        );
```

Comment:
Code would be easier to read with dedicate method `multi_tab.subscribe()` rather than `multi_tab.bus.addEventListener()`